app/views/posts/show.html.erb

```diff
@@ -1,3 +1,23 @@
-<% 2.times do %>
-  <h2>Posts show view</h2>
-<% end %>
\ No newline at end of file
+<main>
+  <section class="post-section">
+    <h2>Information for post</h2>
+    <article class="post-info">
+      <div class="post-card">
+        <h3><%= @post.title %> by user:  <%= @user.name %></h3>
+        <p>Number of comments: <%= @post.comments_counter %> - Number of likes: <%= @post.likes_counter %></p>
+        <p class="post-text"><%= @post.text %></p>
+      </div>
+    </article>
+
+    <article class="comments">
+ 
+        <ul class="total-comments">
+          <% @post.comments.each do |comment| %>
+            <li class="comment"><%= comment.author.name %>: <%= comment.text %></li>
+          <% end %>
+        </ul>
+      
+    </article>
+    
+  </section>
```

Comment:
- [ ] I think it would be better if you add some styling on the `app/views/posts/show.html.erb` view page so it matches the given **[wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_single_post.png?token=GHSAT0AAAAAAB56UIU3TR5AEC7LGBOHXJJUY6S7QCQ)** design. Right now this page has no styling. I know you  can do that

Reply:
Styling is nice to have but focus on making sure everything functions first. (Note: Ignore login logic, you would be using Devise in coming milestone)